gfx/UltimateCataclysm/pngs_normal_32x32/items/corpses/corpses.json

```
@@ -1,12 +1,14 @@
 [
   {
-  "id": "corpse",
-  "fg": "corpse",
-  "bg": ""
+    "id": "corpse",
+    "fg": "corpse"
   },
   {
-  "id": "corpse_generic_human",
-  "fg": "corpse_generic_human",
-  "bg": ""
- }
+    "id": "corpse_generic_human",
+    "fg": "corpse_generic_human"
+  },
+  {
+    "id": "corpse_mon_zombie_child",
+    "fg": "corpse_mon_zombie_child"
+  }
```

```suggestion
  }
```
As usual, don't need the json if fg and id are the same

---

Oops... That's a artifact from my try to see if corpse item variants work or not x)
Gonna delete the whole JSON though.